client/src/sass/ManageProjects.scss

```diff
@@ -313,7 +312,7 @@ textarea {
 }
 
 .event-list-details {
- display: flex;
+  display: flex;
```

Comment:
Can you please drop this change as per other comments in this PR review re: linting

Reply:
Easiest way to distinguish rn between editable content vs only view would be to **make use of the form field when editing** and **have no element around the information when only viewed**
ie:
<img width="299" alt="Screenshot 2023-07-03 at 10 25 08 AM" src="https://github.com/hackforla/VRMS/assets/80308705/08934957-7a22-4962-9f13-f5f6c4862e56">

You'd have to disregard the "[edit]" buttons. Not sure what the feedback on that was but we can revisit the idea of only having one button for editing instead of one for each field. I can have some updated designs for you all later this afternoon